mitodl/salt-ops
# feat: add feature flags for select field

# What are the relevant tickets?
https://github.com/mitodl/ocw-studio/issues/1863

Related PR https://github.com/mitodl/ocw-studio/pull/1888

# Description (What does it do?)
Adds feature flag env var values for for https://github.com/mitodl/ocw-studio/pull/1888 

# How can this be tested?
This will be tested once deployed to RC. Check the Heroku config to verify that new feature flag vars exist.
